docs/how-to-guides/develop-a-flex-flow/class-based-flow.md

```diff
@@ -121,9 +90,33 @@ flow_obj = MyFlow(model_config=config, flow_config={})
 pf.run(flow=flow_obj, data="./data.jsonl")
 ```
 
+## Batch run with YAML
+
+User can batch run a flex flow. Flow init function's param is supported by `init` parameter.
+
+```python
```

Comment:
do we have CLI experience for flex flow? I guess we support, but do not find in the doc

Reply:
Updated